### PR TITLE
DOCS new headings for change log template

### PR DESCRIPTION
**Pull request by brynwhyman**
Adding additional information to the change log template to better match the recent 2.3 release and the suggested changes in https://github.com/silverstripe/cwp/issues/188

### CODE REVIEW

File: .changelog.md.twig [hunk 1]
```diff
@@ -6,27 +6,39 @@ This release includes CMS and Framework version X.X.X.
...
-```
+## Known Issues
+
+* {{ [Known issue](GitHub link) }}
+
+### Expected test failures
+
+The following PHPUnit test failures are expected and do not represent functional issues in CWP:
+
+* {{ [test failure] }}
```
Comment (robbieaverill) [inline .changelog.md.twig]: FYI the `{{ abc }}` syntax has special meaning as a placeholder in Twig templates. I suspect it won't matter unless we define a replacement for them though. May be best to use something else like `*` or `_` anyway

File: .changelog.md.twig [hunk 2]
```diff
@@ -6,27 +6,39 @@ This release includes CMS and Framework version X.X.X.
...
-## Upgrading instructions
+The [release announcement](#) includes the note worthy features, but be sure to review the change log for full detail.
 
-In order to update an existing site to use the new basic recipe the following changes to your composer.json can be made:
+{{ other upgrade notes here }}
 
-```
-...
-```
+## Known Issues
```
Comment (robbieaverill) [inline .changelog.md.twig]:
```suggestion
## Known Issues
```
```suggestion
## Known issues
```

File: .changelog.md.twig [hunk 3]
```diff
@@ -6,27 +6,39 @@ This release includes CMS and Framework version X.X.X.
 
 - [Framework X.X.X](#)
 
-Upgrading to Recipe {{ version }} is optional, but is recommended for all CWP sites.
+Upgrading to Recipe {{ version }} is recommended for all CWP sites. This upgrade can be carried out by any development team familiar with SiliverStripe CMS. However, if you would like SilverStripe's assistance, you can request support via the [Service Desk](https://www.cwp.govt.nz/service-desk/new-request/).
 
-This upgrade can be carried out by any development team familiar with SilverStripe CMS, but if you would like SilverStripe's assistance, please let us know.
+## New Features
```
Comment (robbieaverill) [inline .changelog.md.twig]:
```suggestion
## New features
```

File: .changelog.md.twig [hunk 4]
```diff
@@ -6,27 +6,39 @@ This release includes CMS and Framework version X.X.X.
...
+* {{ [Known issue](GitHub link) }}
+
+### Expected test failures
+
+The following PHPUnit test failures are expected and do not represent functional issues in CWP:
+
+* {{ [test failure] }}
 
-(...other upgrade notes here)
+## Security Considerations
```
Comment (robbieaverill) [inline .changelog.md.twig]:
```suggestion
## Security considerations
```

File: .changelog.md.twig [hunk 5]
```diff
@@ -6,27 +6,39 @@ This release includes CMS and Framework version X.X.X.
...
 
-## Known issues
+This release includes {{ several }} security fixes. Please see the release announcements for more detailed descriptions of each[ but note that the following issues have modified CVSS Environmental scores which take built-in protections from the CWP platform into account]. We highly encourage upgrading your CWP projects to include these security patches nonetheless.
 
+* {{ CVE issue with modified CVSS Environmental score }}, {{ new CVSS score }}
+
+## Upgrading Instructions
```
Comment (robbieaverill) [inline .changelog.md.twig]:
```suggestion
## Upgrading instructions
```

File: .changelog.md.twig (hunk 2 above)
Comment (brynwhyman) [reply .changelog.md.twig]: Actioned

File: .changelog.md.twig (hunk 3 above)
Comment (brynwhyman) [reply .changelog.md.twig]: Actioned

File: .changelog.md.twig (hunk 4 above)
Comment (brynwhyman) [reply .changelog.md.twig]: Actioned

File: .changelog.md.twig (hunk 5 above)
Comment (brynwhyman) [reply .changelog.md.twig]: Actioned

File: .changelog.md.twig (hunk 1 above)
Comment (brynwhyman) [reply .changelog.md.twig]: Thanks for the heads up. Updated to use Twig comment syntax, see https://twig.symfony.com/doc/2.x/templates.html#comments

**Pull request merged**